### DISCUSSION TITLE
Neovim Performance

### DISCUSSION THREAD

**Discussion by jiyometrik** [jdhao/nvim-config#18] [General]
Hi everyone, I'm curious about Neovim's performance for you, as most people seem to have a ton of plugins but load time is less than 1s. And there are others whose load time might as well be 1 hour, with very few plugins.
For myself, I have quite a few (9) plugins, and my load time ranges from 0.8 - 1.5 seconds.
What about you guys? What plugins do you guys use, and what plugin manager? I'd love to find out.

**Comment by jdhao** [comment]
@g-e-o-m-e-t-r-i-c  That is strange. I have over 60 plugins. My load time is about 350 ms on a Linux machine. I would say 800 ms is way too much.

**Reply by jiyometrik** [reply]
Ah, I see. So the problem must be WSL then. (I use WSL because I don't have access to a reliable USB drive to boot Linux from)

**Comment by atulkundaria** [comment]
How can I find all the keybinding in this repository.

**Reply by jdhao** [reply]
Mainly in `core/mappings.vim` and part in `lua/custom-map.lua`.

**Comment by r3isenfe1d** [comment]
Changed the tab width in the `optinons.vim` file as 4 characters, but in the C/C++ files tab is still 2 characters. How to fix it?

**Reply by jdhao** [reply]
see after/ftplugin/cpp.vim. setting there overrides general settings.
